src/main/java/uk/gov/ons/ssdc/exceptionmanager/endpoint/ReportingEndpoint.java

```diff
@@ -82,8 +82,7 @@ public void storeSkippedMessage(@RequestBody SkippedMessage skippedMessage) {
     quarantinedMessage.setHeaders(skippedMessage.getHeaders());
     quarantinedMessage.setMessageHash(skippedMessage.getMessageHash());
     quarantinedMessage.setMessagePayload(skippedMessage.getMessagePayload());
-    quarantinedMessage.setQueue(skippedMessage.getQueue());
-    quarantinedMessage.setRoutingKey(skippedMessage.getRoutingKey());
```

Comment:
shouldn't have removed `routingKey`... that will end up being renamed to `topic`